chess.py

```diff
@@ -0,0 +1,120 @@
+class Figura:
+    color: str = "white"
+# место на доске
+    x: int = 0
+    y: int = 0
+
+    def __init__(self, x1: int, y1: int, color_temp: str):
+        self.set_color(color_temp)
+        if self._check_border(x1, y1):
+            self.x = x1
+            self.y = y1
+        elif color_temp == "white":
+            self.x = 0
+            self.y = 0
+        elif color_temp == "black":
+            self.x = 7
+            self.y = 7
+
+    def set_color(self, color_: str):
+        if color_ == "white":
+            self.color = color_
+        else:
+            self.color = "black"
+
+    def change_color(self):
+        if self.color == "white":
+            self.color = "black"
+        elif self.color == "black":
+            self.color = "white"
+        else:
+            return False
+
+    def _check_border(self, x1: int, y1: int) -> bool:
+        if 0 <= x1 <= 7 and 0 <= y1 <= 7:
```

Comment:
```suggestion
        return 0 <= x1 <= 7 and 0 <= y1 <= 7
```

Те, що тут написано можно спростити як:
```python
if True:
    return True
```
нащо ці зайві маніпуляції.